jobs/fx_usage_report.sh

```diff
@@ -12,4 +12,5 @@ cd Fx_Usage_Report
 python usage_report/usage_report.py \
     --date $date \
     --sample 10 \
-    --output-bucket $bucket
+    --output-bucket $bucket \
+    --output-prefix "$deploy_environment/usage_report_data"
```

Comment:
The intent here is to make it easier to figure out what's writing the data, I'm guessing?

Reply:
Yeah, eventually I'll move to have code first deployed to stage and then to prod. So for now these will both be the same.